README.md

```diff
@@ -151,7 +151,7 @@ CMake can take a few options.
 
 Variable          | Type | Description
 ------------------|------|--------------
-BUILD_SHARED_LIBS | Bool | The default build generates static library. Enable this to generate shared (dll/so) library.
+BUILD_SHARED_LIBS | Bool | The default build generates a static library. Enable this to generate shared (dll/so) library.
```

Comment:
er.. I'm not even sure this description is accurate, and if it is, then it shouldn't be.  The default definitely should not be to just build a static library.

Reply:
By default, cmake will build one or the other. If BUILD_SHARED_LIBS is ON, then cmake will create a shared library, but not a static library. There are ways to change that behavior though, with only minor editing of CMakeLists.txt apparently. Some info about it here: https://stackoverflow.com/questions/2152077/is-it-possible-to-get-cmake-to-build-both-a-static-and-shared-version-of-the-sam